app/models/theme.rb

```diff
@@ -874,7 +877,8 @@ def migrate_settings(start_transaction: true)
         record.calculate_diff(res[:settings_before], res[:settings_after])
         record.save!
       end
-      self.save!
+
+      self.reload
```

Comment:
I'm curious, why did you change this to `reload`?

Reply:
O because there is no need to save here and saving was causing the callbacks to fire and ended up publishing redundant MessageBus messages which was causing a test to fail.

Reply:
Ah, I see. Thanks for the explanation.